src/atoms/TextInput/Label.native.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Platform } from 'react-native';
+import styled from 'styled-components/native';
+import Text from '../Text';
+import View from '../View/View.native';
+import Space from '../Space';
+
+const styles = {
+  container: {
+    paddingTop({ inputLayoutDimensions }) {
+      // For aligning left label to the center of Text Field
+      if (Platform.OS === 'android') {
+        return `${inputLayoutDimensions.height * 0.36}px`;
```

Comment:
`0.36` is OS specific, right? Better to define it in a constant on top.
Also, we need to keep these numbers in mind while handling scaling.

Reply:
Scaling is next sprint :P

Reply:
Yes. But let's keep it in a constant.

Reply:
Yes, commited that already.